corehq/apps/sms/api.py

```diff
@@ -41,6 +41,10 @@ class BackendAuthorizationException(Exception):
     pass
 
 
+def get_utcnow():
```

Comment:
@czue seems to have done it here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/tests/test_pillow.py#L67-L67